### DISCUSSION TITLE
Vscode Extention

### DISCUSSION THREAD

**Discussion by JanLouisP** [almarklein/timetagger#488] [Ideas]
A Vscode extention like wakatime would be awesome, so everytime I open a project it tracks automaticaly the time to its respective Project :)

**Comment by almarklein** [comment]
There is one now: https://marketplace.visualstudio.com/items?itemName=TimeTaggerExtension.tte

**Reply by delize** [reply]
I have recently tried this, but if you move off of the vscode extension, it seems to forget it's tracking or recording state?
